docs/api.md

```diff
@@ -2830,6 +2913,56 @@ Identical to [the non-Draft version](/reference/submissions/attachments/clearing
 + Response 403 (application/json)
     + Attributes (Error 403)
 
+# Group Datasets
+
+_(introduced: version 2022.3)_
+
+A Dataset is a collection of Entities. A Dataset's schema is defined by the Forms that create Entities in that Dataset. See the [ODK XForms specification](https://getodk.github.io/xforms-spec) for defining Datasets in Form definitions.
+
+Entities are added to a Dataset when a Submission of a Form that creates Entities is **approved**.
+
+### Related APIs:
+
+- [Link a Dataset to a Form Attachment](#reference/forms/draft-form/updating-a-draft-form-attachment)
+- [Get a Form's related Datasets](#reference/forms/related-datasets)
+
+
+## Datasets [GET /projects/{projectId}/datasets]
+
+Returns the list of Datasets in a project
+
++ Parameters
+    + projectId: `16` (number, required) - The numeric ID of the Project
+
++ Response 200 (application/json)
+    This is the standard response
+
+    + Attributes (array[Dataset])
+
++ Response 403 (application/json)
+    + Attributes (Error 403)
+
+## Download Dataset [GET /projects/{projectId}/datasets/{name}/entities.csv]
+
+Download Dataset as CSV file
```

Comment:
Add more info here, like what the csv is likely to look like.
We also should mention that the order of properties might change. (order not guaranteed right now)